config/quality/pmd/pmd-ruleset.xml

```diff
@@ -51,7 +51,9 @@
         <exclude name="ShortClassName" />
         <exclude name="VariableNamingConventions" />
     </rule>
-    <rule ref="rulesets/java/optimizations.xml" />
+    <rule ref="rulesets/java/optimizations.xml">
+        <exclude name="RedundantFieldInitializer" />
```

Comment:
This rule complained about [this line](https://github.com/ericcornelissen/NervousFish/pull/194/files#diff-149538cc1f1f5e4370b6ae93d6489330R74). I don't think it is ever bad to explicitly initialize a variable...